lib/bin/check-file-headers.js

```diff
@@ -48,11 +50,21 @@ process.stdin.on('end', () => {
     if (!withConsistentYear.startsWith(header)) {
       console.error(entryPath);
       missing = true;
+
+      if (mode === 'fix') {
+        fs.writeFileSync(entryPath, `${header}\n${contents}`);
+      }
     }
   }
 
   if (missing) {
-    console.error('\nThe files above do not have the expected file header.');
-    process.exit(1);
+    if (mode === 'fix') {
+      console.error('\nThe files above have been changed to include the required header.');
+    } else {
+      console.error('\nThe files above do not have the expected file header.');
+      console.error('\nPlease re-run this script with the --fix flag, or add the following header manually:');
+      console.error(`\n${header}`);
```

Comment:
Similarly, if we're going to display a file header here, I think it'd be good to display the current year.

Reply:
Ah I think this comment is applicable again. Could we use `thisYearHeader` here as well (or do something along those lines)?